### PR TITLE
Relax Setup Requirements & Fix a Bash Job Bug

### PR DESCRIPTION
**Pull request by lichtefeld**

### CODE REVIEW

File: pegasus_wrapper/workflow.py [hunk 1]
```diff
@@ -861,6 +863,7 @@ def run_bash(
 
         commands_with_ckpt = list(command)
         commands_with_ckpt.append(f"touch {ckpt_path.absolute()}")
+        commands_with_ckpt.insert(0, f"cd {job_dir}")
```
Comment (joecummings) [inline pegasus_wrapper/workflow.py:866]: Can you explain what the bug was in the comments?

Comment (lichtefeld) [reply pegasus_wrapper/workflow.py:866]: Matching consistency with how the script which runs the Python jobs are generated. By default, the working directory is just a random directory where Pegasus is executing the job in but by moving the executing back to the location of the `Locator` for the job relative paths based on the Locators can work automatically without a user needing to explicitly convert to absolute paths.

Comment (joecummings) [reply pegasus_wrapper/workflow.py:866]: Awesome!

**Review by joecummings — APPROVED** [review]
lgtm

**Pull request merged**